packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -697,7 +697,7 @@ export class IndexedDbPersistence implements Persistence {
     // we successfully deleted its metadata from IndexedDb.
     this.removeClientZombiedEntry();
     if (deleteData) {
-      await SimpleDb.delete(this.dbName);
+      await IndexedDbPersistence.clearPersistence(this.dbName);
```

Comment:
I believe that we should now be able to delete this code pass and the deleteData argument.